clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/UBamFileStatus.scala

```diff
@@ -0,0 +1,13 @@
+package org.broadinstitute.clio.transfer.model
```

Comment:
After the big rearrangement of files, I think this should go in the `clio-util` subproject, next to `Location`. It can probably just be named `FileStatus` too, and we can use it for all of the metadata types we add later.